.github/workflows/build-docker-image.yml

```
@@ -0,0 +1,82 @@
+            ${{ env.DOCKERHUB_REGISTRY }}/chairemobilite/${{ github.event.repository.name }}-${{ matrix.name }}
+            ${{ env.GITHUB_REGISTRY }}/{{ github.repository }}-${{ matrix.name }}
+          tags: |
+            # Tag as latest for main
+            type=raw,value=latest,enable=${{ github.ref == 'refs/heads/main' }}
+            # Include the branch name as a tag
+            type=ref,event=branch
+
+      - name: Build and push ${{ matrix.name }} images
+        uses: docker/build-push-action@v5
+        with:
+          context: {{ matrix.context }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)